samples/grpc/local-drone-control-java/src/main/java/local/drones/DeliveryEvents.java

```diff
@@ -35,20 +37,15 @@ public static Behavior<ProjectionBehavior.Command> projectionBehavior(
         var eventsBySlicesQuery =
             GrpcReadJournal.create(
                 system,
+                GrpcQuerySettings.create(system).withInitialConsumerFilter(
+                    // location id already is in the format of a topic filter expression
+                    Arrays.asList(
+                            new ConsumerFilter.ExcludeRegexEntityIds(Collections.singleton(".*")),
```

Comment:
We could use the new ExcludeAll (in all places where we use `.*`)

Reply:
Ah, I had forgotten that one, I'll replace this. 👍